code/modules/research/designs/autolathe_designs.dm

```diff
@@ -1033,6 +1033,14 @@
 	build_path = /obj/item/vending_refill/custom
 	category = list("initial", "Misc")
 
+/datum/design/custom_vendor_board
```

Comment:
I feel like this should be available from service tech fab, but well, we'll see how this works first... just leaving my opinion here.